microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -291,47 +291,55 @@ To use the Fixed Version distribution mode:
         Debug.WriteLine($"FixedRuntime path [{fixedPath}]");
         Environment.SetEnvironmentVariable("WEBVIEW2_BROWSER_EXECUTABLE_FOLDER", fixedPath);
         ```
+
 1. Package and ship the Fixed Version binaries with your app.  Update the binaries as appropriate.
 
-    * For WinUI this might include opening the project file (.csproj) in a code editor, and then add the following code snippet within the project tags:
+    *  For WinUI, this might include opening the project file (`.csproj`) in a code editor, and then add the following code within the project tags:
 
-        ```xml
-        <Content Include="FixedRuntime\(version number)\\**\*.*">
-        <CopyToOutputDirectory>PreserveNewest<CopyToOutputDirector>
-        </Content>
-        ```
+       ```xml
+       <Content Include="FixedRuntime\(version number)\\**\*.*">
```

Comment:
```suggestion
       <Content include="FixedRuntime\(version number)\\**\*.*">
```

Reply:
This change is likely correct, because there's a basic error in the element tags nearby and because attr name is almost(?) always lowercase.

Reply:
@jasonstephen15 This might be uppercase, can you verify?

Reply:
Casing of elements: https://www.w3schools.com/html/html_elements.asp - find "case".
Casing of attributes: https://www.w3schools.com/html/html_attributes.asp - find "case"